sample/confidential_client_certificate_sample.py

```diff
@@ -42,27 +43,35 @@
 
 config = json.load(open(sys.argv[1]))
 
-# Create a preferably long-lived app instance which maintains a token cache.
-app = msal.ConfidentialClientApplication(
+# If for whatever reason you plan to recreate same ClientApplication periodically,
+# you shall create one global token cache and reuse it by each ClientApplication
+global_token_cache = msal.TokenCache()  # More options in https://msal-python.readthedocs.io/en/latest/#tokencache
```

Comment:
Maybe a better name is `in_memory_token_cache`

Reply:
But, my understanding is that this round of effort focuses on "do static caching (i.e. shared per process)" (which was quoted from "[Feature Request] Improve default token caching #602").

The term "global" is closer to "static" in terms of multiple CCA in the same process will share such a cache. The term "in memory" is not. There can be many CCA, each has their own default in-memory cache, but not shared with each other.

Besides, when/if app developer would choose to use a persisted token cache, the line can still be `global_token_cache = msal.SerializableTokenCache()` (I plan to also change the other sample here to use `global_token_cache`).